### PR TITLE
Add expm1 test case to cover the k==1024 case

### PR DESCRIPTION
**Pull request by lygstate**
Refer to https://github.com/jerryscript-project/jerryscript/blob/8e83638daa24b9f9ffffc83e61d55063e2e9d490/jerry-math/expm1.c#L268

JerryScript-DCO-1.0-Signed-off-by: Yonggang Luo luoyonggang@gmail.com

### CODE REVIEW

**Comment by akosthekiss** [comment]
What is 0x1p1023 and why does it need extra coverage? Furthermore, why is the negated value not tested?

**Comment by lygstate** [comment]
> What is 0x1p1023 and why does it need extra coverage? Furthermore, why is the negated value not tested?

Refer to https://github.com/jerryscript-project/jerryscript/blob/8e83638daa24b9f9ffffc83e61d55063e2e9d490/jerry-math/expm1.c#L268

File: tools/unit-tests/gen-test-math.c [hunk 1]
```diff
@@ -404,6 +404,7 @@ main (int argc, char **args)
   GEN_DBL_TEST (expm1 (INFINITY));
   GEN_DBL_TEST (expm1 (-INFINITY));
   GEN_DBL_TEST (expm1 (NAN));
+  GEN_DBL_TEST (expm1 (7.095e+02));
```
Comment (akosthekiss) [inline tools/unit-tests/gen-test-math.c:407]: move this one line down, between 708 and 710 (and re-generate the .inc.h)

**Review by akosthekiss — CHANGES_REQUESTED** [review]
In addition to the change requested in the code, please, update commit title and message as well.

Current commit title is misleading, as it is hard to cover a literal/number, unless it is the input value to the function. But it is not the case here. The test case aims at covering the k==1024 case. So, the title should read "Add expm1 test case to cover the k==1024 case".

As for the commit message, don't include the discussion in there. Use comments to answer questions. The commit message should describe what has been done, and why. But rarely something like "refer to ..."

**Comment by akosthekiss** [comment]
BTW, kind of answer-to-self: 0x1p1023 is a hexadecimal floating point literal with the value of `0x1 * 2**1023`. I haven't used them, like ever.

**Review by akosthekiss — APPROVED** [review]
LGTM

**Review by rerobika — APPROVED** [review]

**Pull request merged**